### ISSUE TITLE
Dragonns for drug discovery?

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepgene#4]
Dragonn (http://kundajelab.github.io/dragonn/) is the most mature deep learning library for genomic data. Would it be possible to adapt dragonn models for drug discovery?

**Comment by vsomnath** [comment]
As far as as I recall, DeepChem still doesn't have full functionality of dragonn ported in right?

There are also some custom layers, a few of which I found in Avanti's [paper](https://www.biorxiv.org/content/10.1101/103663v1) and few from the [Janggu library](https://github.com/BIMSBbioinfo/janggu), which could be ported into DeepGene pr `dc.genomics`.

With TF 2.0 based on tf.keras and tf.contrib going away, it might be easier to integrate DragoNN now. 
Relevant discussion [#1461](https://github.com/deepchem/deepchem/issues/1461).

What do you think?

**Comment by rbharath** [comment]
Yes, agreed. We're going to start pushing to integrate DeepChem layers with tf.keras layers so we get full TF 2.0 inter-op. This will be the next big push after DeepChem 2.2 is cut.
